doc/changelog/pendings/changes_1624109504178983100.md

```diff
@@ -0,0 +1,8 @@
+### Added
+- new hook `CanSendJoinMessageArenaQueue` and `CanSendExitMessageArenaQueue`
+
+### Changed
+- Rename `CanExitJoinMessageArenaQueue` to `CanSendExitMessageArenaQueue`
+
+### How to upgrade
+- Just rename all hooks from `CanExitJoinMessageArenaQueue` to `CanSendExitMessageArenaQueue`
```

Comment:
```suggestion
- Just rename all hooks from `CanExitJoinMessageArenaQueue` and `CanSendMessageArenaQueue`, to `CanSendExitMessageArenaQueue`
```

Reply:
The original hook was called CanSendMessageArenaQueue, some modules are still using that one